### PR TITLE
Add Python Stylechecker

### LINKED ISSUE #32: Add Python Linter to CI
Add PEP 8 style checker to CI build for .py files.

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
Fixes #32

### CODE REVIEW

**Comment by Ian-Stewart-Binks** [comment]
@Anirudh94 Please review and merge.

File: Gruntfile.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ module.exports = function(grunt) {
     grunt.initConfig({
         eslint: {
             all: ['*.js', './public/js/*']
+        },
+        flake8: {
+            src: ['*.py', './']
```
Comment (emmettu) [inline Gruntfile.js]: It's in the util folder

**Comment by Anirudh94** [comment]
INFO: depends on PR #30 being merged

File: Gruntfile.js (hunk 1 above)
Comment (Anirudh94) [inline Gruntfile.js]: `'*.py'` and `'./'` both flake .py files in root only I believe. Change to `['*.py', './util/*.py']` after #30 is in. You can also do `['**/*.py']` which will flake ALL .py files in the project[1]. 
[1] https://www.npmjs.com/package/grunt-flake8#default-options

**Comment by Ian-Stewart-Binks** [comment]
@Anirudh94 Why does it depend on #30?

File: Gruntfile.js (hunk 1 above)
Comment (Ian-Stewart-Binks) [inline Gruntfile.js]: Actually - yeah, it lints _node_modules_...

**Comment by Anirudh94** [comment]
@Ian-Stewart-Binks Well not strictly speaking, but I'd like to merge it after so it makes more sense in commit history since we will actually have a python file to check :)

**Pull request merged**